modules/update-restricted.adoc

```diff
@@ -7,6 +7,12 @@
 
 Update the restricted network cluster to the {product-title} version that you downloaded the release images for.
 
+[NOTE]
+====
+// If you have xref FIXME ../update_service/index.adoc[a local OpenShift Update Service], you can update by using the connected xref FIXME ../updating/updating-cluster.adoc[web console] or xref FIXME ../updating/updating-cluster-cli.adoc[CLI] instructions.
+This section explains the update procedure in the absence of an available Update Service.
+====
+
```

Comment:
Do you think a separate section is needed to explain the update procedure with a local Update Service? e.g. The procedure to get the Update Service route url and the procedure to update the upstream and channel in CVO, etc.

Reply:
I fixed the FIXME links so if they have a local update service, or want to install one, the corresponding referenced section for that should be inclusive.

Reply:
It's helpful. If users want to use a local update service, they have to configure the CVO upstream to use it, right? If that's true, it would be better to document the procedure.

Reply:
Added new section Configuring the Cluster Version Operator (CVO).